Sources/EmbraceCoreDataInternal/CoreDataWrapper.swift

```diff
@@ -104,17 +107,41 @@ public class CoreDataWrapper {
         }
     }
 
+    /// Synchronously performs the given block on the current context.
+    /// This will also create a background task to perform the operation.
+    /// If the background task can't be created, the block will be called without a context.
+    public func performOperation(name: String, _ block: (NSManagedObjectContext?) -> Void) {
+
+        if isTesting {
```

Comment:
You're not actually testing if you do things totally differently than what you do in prod.